src/VideoRemise/VideoRemise/ConfigPage.xaml

```diff
@@ -4,101 +4,146 @@
-            <ComboBox x:Name="videoFeedLeft" Width="890" FontSize="14" Header="Left Camera:" 
-                  PlaceholderText="Pick a camera" Margin="5,0,0,0" IsEnabled="False">
-                <x:String>dummy</x:String>
-            </ComboBox>
-            <ComboBox x:Name="videoFeedCenter" Width="890" FontSize="14" Header="Center Camera:" 
-                  PlaceholderText="Pick a camera" Margin="5,0,0,0">
-                <x:String>dummy</x:String>
-            </ComboBox>
-            <ComboBox x:Name="videoFeedRight" Width="890" FontSize="14" Header="Right Camera:" 
-                  PlaceholderText="Pick a camera" Margin="5,0,0,0" IsEnabled="False">
-                <x:String>dummy</x:String>
-            </ComboBox>
-            <!--<CheckBox x:Name="audioToggle" Margin="5,30,0,10" FontSize="14">Audio feed?</CheckBox>-->
-            <TextBlock Text="Timing and Replay Configuration" FontSize="18" FontWeight="Bold" Margin="5,10,0,0"/>
-            <TextBlock Text="All times in milliseconds" FontSize="14" Margin="5,0,0,0"/>
-            <Grid Margin="5,0,0,0">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition />
-                    <ColumnDefinition />
-                    <ColumnDefinition />
-                </Grid.ColumnDefinitions>
-                <Grid.RowDefinitions>
-                    <RowDefinition />
-                    <RowDefinition />
-                    <RowDefinition />
-                    <RowDefinition />
-                </Grid.RowDefinitions>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="1" Grid.Column="0">
-                    <TextBlock Text="Epee" FontSize="14"/>
-                </Border>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="2" Grid.Column="0">
-                    <TextBlock Text="Foil" FontSize="14"/>
-                </Border>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="3" Grid.Column="0">
-                    <TextBlock Text="Saber" FontSize="14"/>
-                </Border>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="0" Grid.Column="1" >
-                    <TextBlock Text="Pre-Trigger" FontSize="12" HorizontalAlignment="Center"/>
-                </Border>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="0" Grid.Column="2">
-                    <TextBlock Text="Post-Trigger" FontSize="14" HorizontalAlignment="Center"/>
-                </Border>
-                <Border BorderThickness="1" BorderBrush="Black" Grid.Row="1" Grid.Column="1">
-                    <TextBox x:Name="epeePre" Text="0" FontSize="12" HorizontalAlignment="Center" 
+        <StackPanel Width="Auto">
+            <controls:Expander IsExpanded="True">
+                <controls:Expander.Header>
+                    <TextBlock Text="Scoring Machine Configuration" FontSize="16" FontWeight="Bold" HorizontalAlignment="Left"/>
```

Comment:
It would be nice eventually if the expanding sections were the same size when not expanding. And if they were centered (along with the ok and cancel buttons).

Reply:
Agreed. I figured it out.